ols/src/llms/providers/provider.py

```diff
@@ -44,6 +66,33 @@ def __init__(
         self.model = model
         self.provider_config = provider_config
         self.params = self._override_params(params or {})
+        # self._validate_parameters()
+
+    def _validate_parameters(self) -> None:
+        """Validate parameters for LLM provider."""
+        provider = self.provider_config.type
+
+        if provider is None:
+            logger.warning("Provider type is not set")
+            return
+
+        if provider not in available_provider_parameters:
+            logger.warning(
+                f"Available parameters for provider {provider} are not setup."
```

Comment:
> f"Available parameters for provider {provider} are not defined. Parameters validation is disabled."

Or something like that to tell the user what is actually the outcome of that warning.

Reply:
ok that makes sense, thank you @onmete

Reply:
updated